sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -48,10 +48,15 @@ export default defineComponent({
     );
 
     const getAwardValue = (awardType: string): string | number => {
-      return (
-        props.awardDetails[`${props.identifier}${awardType.toLowerCase()}`] ??
-        "(Not eligible)"
-      );
+      const awardValue =
+        props.awardDetails[`${props.identifier}${awardType.toLowerCase()}`];
+      // If the award is defined but no values are present it means that a receipt value is missing.
```

Comment:
Please clarify, if the award is requested and the value is not present in the receipt, it means its not approved or it will be populated in the future receipts as it is missed in the current receipt? The comments makes me understand the later.

Reply:
There isn't so far anything that indicates that multiple receipts will be received. One receipt record is expected for one disbursement.
This comment alone is not intended to explain all the context for the entire feature but I hope the comments along the implementation would help the understanding overall picture.
Please let me know if further clarifications are needed.